aws_adfs/login.py

```diff
@@ -133,11 +134,11 @@
 )
 @click.option(
     "--duo-factor",
-    help=f'Use a specific Duo factor, overriding the default one configured server side. Known Duo factors that can be used with aws-adfs are "{DUO_UNIVERSAL_PROMPT_FACTOR_DUO_PUSH}", "{DUO_UNIVERSAL_PROMPT_FACTOR_PHONE_CALL}", and "{DUO_UNIVERSAL_PROMPT_FACTOR_WEBAUTHN}". Only supported with Duo Universal Prompt.',
+    help=f'Use a specific Duo factor, overriding the default one configured server side. Known Duo factors that can be used with aws-adfs are "{DUO_UNIVERSAL_PROMPT_FACTOR_DUO_PUSH}", "{DUO_UNIVERSAL_PROMPT_FACTOR_PHONE_CALL}", "{DUO_UNIVERSAL_PROMPT_FACTOR_WEBAUTHN}" and "{DUO_UNIVERSAL_PROMPT_FACTOR_PASSCODE}". Only supported with Duo Universal Prompt.',
```

Comment:
Isn't the `Only supported with Duo Universal Prompt` part no longer valid with this PR too?
```suggestion
    help=f'Use a specific Duo factor, overriding the default one configured server side. Known Duo factors that can be used with aws-adfs are "{DUO_UNIVERSAL_PROMPT_FACTOR_DUO_PUSH}", "{DUO_UNIVERSAL_PROMPT_FACTOR_PHONE_CALL}", "{DUO_UNIVERSAL_PROMPT_FACTOR_WEBAUTHN}" and "{DUO_UNIVERSAL_PROMPT_FACTOR_PASSCODE}".',
```